### ISSUE TITLE
Countdown frames not displayed on first occurence

### ISSUE THREAD

**Issue by PatTheMav** [AzeriteTeam/AzeriteUI#19]
## The problem  
After login, the first time I participate in a Battleground, Island Expedition or War Front, the countdown sounds can be heard, but the timer as well as the big numbers in the center of the screen are not visible.

They do appear on all consecutive events during the same session.

### Make sure you can check all these boxes  
- [x] I have the latest addon version available from CurseForge/Twitch.
- [x] I restarted the entire game after updating the addon, not just relogged or reloaded. 

### Does this happen with NO other addons enabled?  
Will update once checked specifically.
- [ ] Yes
- [ ] No

### What classes does it happen on?  
Only have a high-level priest, will see if I can replicate with low-level alts.
- [ ] Death Knight 
- [ ] Demon Hunter 
- [ ] Druid 
- [ ] Hunter 
- [ ] Mage 
- [ ] Monk 
- [ ] Paladin 
- [x] Priest 
- [ ] Rogue 
- [ ] Shaman 
- [ ] Warlock 
- [ ] Warrior 

### Any logs, error output, etc?  
No errors visible.

### Steps to reproduce  
* Log in
* Queue for island expedition
* Wait for the queue to pop
* Wait on the boat and wait for the countdown to start

### Expected behaviour  
* Built-in countdown animation should appear

### Actual behaviour  
* No countdown animation visible on first occurence

**Comment by goldpaw** [comment]
I've noticed this too, but do so few instances I thought it was random. Good catch! 

I don't hide any of this on purpose, but I'm expecting some other hidden frame is affecting the first occurence, something about it not being cached up and therefore the animation blocked or something. The wow texture animation system can be kind of tricky. 

Will look into it!

**Comment by PatTheMav** [comment]
Yeah I though about some event hook missing on first occurence, which is bound only after the first time. Haven't worked with the API in quite some time.. 😂
